Documentation/Background Indexing.md

```diff
@@ -0,0 +1,42 @@
+# Background Indexing
+
+Background indexing in SourceKit-LSP is available as an experimental feature. This guide shows how to set up background indexing and which caveats to expect.
+
+> [!WARNING]
+> Background indexing is still an experimental feature and still has some non-trivial limitations and bugs (see the known issues).
+
+## Set Up
+
+Background indexing is still under active, rapid development, so please build SourceKit-LSP from the `main` branch using the following commands. Things are changing rapidly, so rebuilding daily is not a bad practice.
+
+```bash
+$ git clone sourcekit-lsp
+$ swift package update
+$ swift build -c release
```

Comment:
Is there meant to be a proper URL for the `git clone`? Also would it be worth adding a `cd`? Or is this more meant as pseudocode?

Reply:
Was meant to be proper code. Just … didn’t check it.